### PR TITLE
chore: Update searchgov, deprecate Netlify CMS, update deps

### PR DESCRIPTION
**Pull request by apburnes**
Closes https://github.com/cloud-gov/private/issues/760

## Changes proposed in this pull request:
- Updates to send search requests to search.gov account
- Removes the deprecated Netlify CMS
- Updates dependencies to the latest Gatsby

## Security considerations

Updates deps to latest versions

### CODE REVIEW

**Comment by drewbo** [comment]
@apburnes The preview build looks fine but the query sent to search.gov ends up at https://www.usa.gov/search-error. Should this work or does it require non-template config?

**Comment by apburnes** [comment]
@drewbo I updated the affiliate and access values in the README and gatsby-config to be generic names to represent the agency's search.gov account info

**Review by svenaas — DISMISSED** [review]
I don't see any problems here.

**Comment by apburnes** [comment]
Updated the README to change Federalist references to Pages.

File: README.md
Comment (svenaas) [inline README.md:1]: Ha! Yeah, that's a lot better.

**Review by svenaas — APPROVED** [review]

**Pull request merged**